### DISCUSSION TITLE
Light entity with support for multiple colors

### DISCUSSION THREAD

**Discussion by ShadyMCShadesen** [home-assistant/frontend#11553] [Other feature requests] [answered]
The light entity currently has just one selectable color. But WLED for example supports up to 3 different colors, which cannot be seen or changed in the light entity therefore, saving and setting via the scene function does not work properly.
It would be awesome if the light entity could support up to x amounts of different colors the user can select from. If x could be adapted to the respective area of ​​application, that would also be very helpfull.

**Comment by spacegaier** [comment, marked as answer]
That is a feature request for the backend, since the frontend can only show / support whatever the backend implements. Changes to the light entity therefore need to start in the backend. Please raise a feature request on the community forums for that.
